src/main/java/org/example/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package org.example.bookstore.service.impl;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.example.bookstore.model.Book;
+import org.example.bookstore.repository.BookRepository;
+import org.example.bookstore.service.BookService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class BookServiceImpl implements BookService {
```

Comment:
Missing `CommandLineRunner` bean provider method in main class where you test this class's methods, check task description. But it won't work without the main `application.properties`, which is another reason to test your code manually 😉

Reply:
Resolved, miss to add commandLineRunner after refactoring.
Tested manually:
![image](https://github.com/VolandevlodD/book-store/assets/124697095/66e0e4b5-a515-4359-b31a-8e9d779f8483)